integration-tests/Cargo.toml

```diff
@@ -32,7 +32,7 @@ serde_json = "1"
 testcontainers = { version = "0.15", features = ["experimental"] }
 tokio = { version = "1.28", features = ["full"] }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
-near-workspaces = "0.8.0"
+near-workspaces = { git = "https://github.com/near/near-workspaces-rs.git", branch = "main" }
```

Comment:
why?

Reply:
workspaces 0.8.0/0.9.0 depends on NEAR SDK 4.x which is no longer compilable due to a yanked dependency. See https://github.com/near/near-sdk-rs/issues/1119.

So we have to use the unreleased version of workspaces for now